docs/intro.rst

```diff
@@ -46,3 +48,14 @@ Other future additions we are currently implementing are:
 * full HEASARC-compatible mission support
 * pulsar searches with :math:`H`-test
 * binary pulsar searches
+
+Presentations
+=============
+
+Members of the Stingray team have given a number of presentations which introduce Stingray.
+These include:
+
+- `European Week of Astronomy and Space Science (2018) <http://ascl.net/wordpress/2018/05/24/software-in-astronomy-symposium-presentations-part-3/>`_
```

Comment:
A few more:
https://ui.adsabs.harvard.edu/abs/2020ASPC..522..521M/abstract
https://ui.adsabs.harvard.edu/abs/2019HEAD...1711502S/abstract
Please add at least one, or it almost seems like Stingray wasn't presented in the last three years :)

Reply:
https://speakerdeck.com/abigailstev/stingray-open-source-spectral-timing-software

Reply:
> https://speakerdeck.com/abigailstev/stingray-open-source-spectral-timing-software

I think those are HEAD16, not 17 (confusingly, HEAD16 took place in 2017, and HEAD17 in 2019...!)

Reply:
Ooops sorry. Let us merge this as it is, then we'll open a new PR to make further fixes, if you agree

Reply:
Just pushed a trivial fix to the markup. If you agree with that change, this is good to merge as far as I'm concerned!